airflow/contrib/sensors/emr_step_sensor.py

```diff
@@ -16,52 +16,102 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
 from airflow.contrib.hooks.emr_hook import EmrHook
 from airflow.contrib.sensors.emr_base_sensor import EmrBaseSensor
 from airflow.utils.decorators import apply_defaults
 
 
 class EmrStepSensor(EmrBaseSensor):
     """
-    Asks for the state of the step until it reaches a terminal state.
+    Asks for the state of the step until it reaches any of the target states.
     If it fails the sensor errors, failing the task.
 
+    With the default target states, sensor waits step to be completed.
+
     :param job_flow_id: job_flow_id which contains the step check the state of
     :type job_flow_id: str
+
```

Comment:
We rather do not use new line here

Reply:
Some of the classes has this, and it seems better to me. But if you want so, I will remove all of the empty line between param-type groups.